README.md

```diff
@@ -154,6 +154,19 @@ If you're looking for a good paper on online hate training datasets (beyond our
 * Medium: Text 
 * Reference: Davidson, T., Warmsley, D., Macy, M. and Weber, I., 2017. Automated Hate Speech Detection and the Problem of Offensive Language. ArXiv,. 
 
+#### Exploring Hate Speech Detection in Multimodal Publications
+* Link to publication: [https://gombru.github.io/2019/10/09/MMHS/] (https://drive.google.com/file/d/1S9mMhZFkntNnYdO-1dZXwF_8XIiFcmlF/view)
```

Comment:
this markdown link is broken, there's a space between the [] () where it should look like []()

Reply:
Leon - I just went back over the hsp.com and this entry actually already exists on the website